src/promptflow-core/promptflow/integrations/parallel_run/__init__.py

```diff
@@ -0,0 +1,6 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from .processor import ParallelRunProcessor, create_processor
```

Comment:
Since it is only used for component, maybe it should be part of pfazure?

Reply:
current pfazure is promptflow azure devkit actually, I don't think runtime image will install it. As offline synced, we can add a new `promptflow-parallel` like [promptflow-eval](https://github.com/microsoft/promptflow/commit/078549534a3685d45cc2c8e29263c4eb6a7ca2ce) and talk to runtime team for in which image we should install this standalone package

Reply:
moved parallel processing code to promptflow-parallel